osu.Game/Screens/Play/HUD/HitErrorMeters/HitErrorMeter.cs

```diff
@@ -34,6 +37,9 @@ protected override void LoadComplete()
         {
             base.LoadComplete();
 
+            if (player != null)
+                player.OnSeek += Clear;
```

Comment:
This is a reference memory leak. For instance, if switching skins or using the skin layout editor, references to each of the `HitErrorMeter`s created along the way are going to be held by `player.OnSeek` events.